pkg/compiler/codegen.go

```diff
@@ -2010,6 +2010,32 @@ func (c *codegen) convertBuiltin(expr *ast.CallExpr) {
 		for range len(expr.Args) - 1 {
 			emit.Opcodes(c.prog.BinWriter, opcode.MAX)
 		}
+	case "clear":
+		switch typ := c.typeOf(expr.Args[0]).(type) {
+		case *types.Map:
+			emit.Opcodes(c.prog.BinWriter, opcode.CLEARITEMS)
+		case *types.Slice:
+			emit.Opcodes(c.prog.BinWriter, opcode.DUP, opcode.SIZE, opcode.DEC,
```

Comment:
Need a test-case for empty slice, I need to ensure that it works properly with `-1` returned by `DEC`.

Reply:
Also for an empty map.